cosmos/config/targets/INST/cmd_tlm_server.txt

```diff
@@ -6,3 +6,4 @@ INTERFACE INST_INT simulated_target_interface.rb sim_inst.rb
   PROTOCOL READ OverrideProtocol
   # This will ignore all received ADCS packets
   #PROTOCOL READ IgnorePacketProtocol INST ADCS
+  DONT_CONNECT
```

Comment:
just adding this here so it doesn't auto-connect on the dev server, where we don't support this interface